models/api_host.go

```diff
@@ -94,6 +97,6 @@ func (a *ApiHost) ConvertAPIHostToNMHost(currentHost *Host) *Host {
 	h.IsDefault = a.IsDefault
 	h.NatType = currentHost.NatType
 	h.TurnEndpoint = currentHost.TurnEndpoint
-
+	h.PersistentKeepalive = time.Duration(a.PersistentKeepalive)
```

Comment:
@gabrielseibel1 i think this is the reason for the nanoseconds. should be `a.PersistentKeepalive * time.Second`

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/4828648a8a07d4f0bfd5ce85c56c0e3afc3bfe98